lib/improver/utilities/cube_manipulation.py

```diff
@@ -444,7 +465,7 @@ def _equalise_coord_bounds(cubes):
     """
     # Check each cube against all remaining cubes
     for i, this_cube in enumerate(cubes):
-        for later_cube in cubes[i+1:]:
+        for later_cube in cubes[i + 1:]:
```

Comment:
Nope, could you revert this please.
(See these points about when and when not to use spaces around operators: https://www.python.org/dev/peps/pep-0008/#other-recommendations)

Reply:
Fixed.